plugin-build/src/test/kotlin/io/sentry/android/gradle/SentryPluginTest.kt

```diff
@@ -272,8 +272,9 @@ class SentryPluginTest(
             .appendArguments(":app:assembleDebug", "--info")
             .build()
 
+        val instr = "OkHttpEventListener, OkHttp"
```

Comment:
hm, has this changed 🤔 ?

Reply:
in theory this test should've failed on `main` as well.

Reply:
yeah, the problem is that it never finishes the CI run but aborts lol. good catch anyway